packages/js/components/src/experimental-select-tree-control/select-tree.tsx

```diff
@@ -131,7 +132,7 @@ export const SelectTree = function SelectTree( {
 			setInputValue( event.target.value );
 		},
 		placeholder,
-		value: inputValue,
+		value: decodeEntities( inputValue ),
```

Comment:
Is this needed? I reverted this change and things still appear to work as expected.

Reply:
Turns out it's not. This is the input field that is typed upon when e.g. you're typing on the Category field. I will remove it.